src/main/java/com/parasoft/findings/jenkins/coverage/api/metrics/charts/CoverageTrendChart.java

```diff
@@ -71,10 +71,13 @@ public LinesChartModel create(final Iterable<BuildResult<CoverageStatistics>> re
             LineSeries lineSeries = new LineSeries(Messages.Metric_LINE(),
                     LINE_COVERAGE_COLOR, StackedMode.SEPARATE_LINES, FilledMode.FILLED,
                     dataSet.getSeries(CoverageSeriesBuilder.LINE_COVERAGE));
+            double maxCoverageValue = Math.ceil(dataSet.getMaximumValue() / 5) * 5;
```

Comment:
I have tried to use a set of build with small changed coverage:
This is the result with multiple of 5:
![image](https://github.com/jenkinsci/parasoft-findings-plugin/assets/109122571/a54d36f1-2afe-4762-8db8-0dec4efb68c3)
This is the result with multiple of 2:
![image](https://github.com/jenkinsci/parasoft-findings-plugin/assets/109122571/1f3262a2-2329-4df3-ae75-7106a471ff40)

Reply:
I believe the deviation is still not noticeable enough. To make the chart useful, could we try calculating the space as follows to see the result:
`Math.ceil(maxVal - minVal)
`

Please update the failed unit tests later.

Reply:
![Has_Samll_Changes](https://github.com/jenkinsci/parasoft-findings-plugin/assets/109123224/74150e74-39fa-4bff-b0ba-e0c78440bcc5)
![Has_Large_Changes](https://github.com/jenkinsci/parasoft-findings-plugin/assets/109123224/41ddc89b-9363-48e3-9549-c756639a8756)
When the coverage value changes

![No_Changes](https://github.com/jenkinsci/parasoft-findings-plugin/assets/109123224/a2799246-1099-4141-8384-13973fa19486)
When coverage does not change

![Change_Extremely_After_Maintain](https://github.com/jenkinsci/parasoft-findings-plugin/assets/109123224/b6fe0ec9-2ae6-4470-8b8f-c18658a90bb3)
When coverage changes a lot after stable status